### DISCUSSION TITLE
installing version  covsirphy 2.20.2

### DISCUSSION THREAD

**Discussion by jparra0419** [lisphilar/covid19-sir#799] [General]
I followed the advise of lisphilar to use the last version of  "covsirphy".

I tried: pip install covsirphy --upgrade; also pip install covsirphy==2.20.2
and I got:
ERROR: Could not find a version that satisfies the requirement covsirphy==2.20.2 (from versions: 2.3.0, 2.3.1, 2.3.2, 2.4.0, 2.4.1, 2.4.2, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.5.4, 2.6.0, 2.7.0, 2.7.1, 2.7.2, 2.7.3, 2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.9.0, 2.9.1, 2.10.0, 2.11.0, 2.12.0, 2.13.0, 2.13.1, 2.13.2, 2.13.3, 2.14.0.
Furthermore I tried:
pip install --upgrade git+https://github.com/lisphilar/covid19-sir.git#egg=covsirphy
and I get the same ERROR.  I would appreciate some help from the experts ---Thank you

**Comment by lisphilar** [comment]
This will be discussed in new issue #800.
